### DISCUSSION TITLE
How to trigger SR04 on demand with Nodemcu

### DISCUSSION THREAD

**Discussion by shashwat001** [arendst/Tasmota#20008] [Q&A] [answered]
I have configured sr04 in tasmota with gpio as follows
`stat/ultrasonicSensor/RESULT = {"GPIO0":{"0":"None"},"GPIO1":{"0":"None"},"GPIO2":{"0":"None"},"GPIO3":{"0":"None"},"GPIO4":{"0":"None"},"GPIO5":{"0":"None"},"GPIO9":{"0":"None"},"GPIO10":{"0":"None"},"GPIO12":{"1888":"SR04 Ech/RX"},"GPIO13":{"0":"None"},"GPIO14":{"1856":"SR04 Tri/TX"},"GPIO15":{"0":"None"},"GPIO16":{"0":"None"},"GPIO17":{"0":"None"}}`

This keeps finding distance at certain interval. However I want to get the distance on demand (from openhab). Can anyone suggest how can I configure module for that?

**Comment by sfromis** [comment, marked as answer]
Needs no Tasmota configuration. If you issue the command `Status 10` to the Tasmota device, the result includes sensor values. I already suggested that on Discord.
